eslint.config.mjs

```diff
@@ -228,173 +229,47 @@ export default typescript.config([
    */
   {
     name: 'eslint/rules',
+    // https://eslint.org/docs/latest/rules/
     rules: {
```

Comment:
I removed rules in this list if the are:
- disabled here, and by default in https://eslint.org/docs/latest/rules/
- enabled here and enabled in https://github.com/eslint/eslint/blob/main/packages/js/src/configs/eslint-recommended.js